crates/pixi_config/src/lib.rs

```diff
@@ -445,6 +447,26 @@ impl From<ConfigCliPrompt> for Config {
         config
     }
 }
+#[cfg(feature = "rattler_repodata_gateway")]
+impl From<Config> for rattler_repodata_gateway::ChannelConfig {
+    fn from(config: Config) -> Self {
+        let default_source_config = config
+            .repodata_config
+            .as_ref()
+            .map(|config| SourceConfig {
+                jlap_enabled: !config.disable_jlap.unwrap_or(false),
+                zstd_enabled: !config.disable_zstd.unwrap_or(false),
+                bz2_enabled: !config.disable_bzip2.unwrap_or(false),
+                cache_action: Default::default(),
+            })
+            .unwrap_or_default();
+
+        rattler_repodata_gateway::ChannelConfig {
+            default: default_source_config,
+            per_channel: Default::default(),
+        }
+    }
+}
```

Comment:
This is a duplicate of the existing `from_pixi_config` function.

Reply:
forgot to remove it! I want to keep it's implementation in this trait, so when you need to convert you just 
call config.into()
wdyt?